R/aes.r

```diff
@@ -158,7 +158,7 @@ print.uneval <- function(x, ...) {
 #' @export
 standardise_aes_names <- function(x) {
   # convert US to UK spelling of colour
-  x <- sub("color", "colour", x, fixed = TRUE)
+  x <- sub("^color$", "colour", x, fixed = TRUE)
```

Comment:
```suggestion
  x <- sub("^color$", "colour", x)
```

or simply

``` r
x[x == "color"] <- "colour"
```

Reply:
I've decided on the alternative approach, as we actually rely on the arbitrary color renaming

Reply:
Sure.